python/ops.py

```diff
@@ -295,37 +434,48 @@ def splat_non_uniform(x, bins):
   method takes NumPy arrays instead of TF tensors, and returns a Numpy array.
 
   Args:
-    x: A 1D vector of values being splatted, in the shape of [batch_size].
+    x: A 2D matrix of values being splatted, in the shape of [batch_size, 1].
     bins: 1D vector, in the shape of [N], where N is the number of bins.
 
```

Comment:
Can you give a simple example of this function in action in the comment?

Reply:
Added: 
  Example: splat_non_uniform(tf.convert_to_tensor([0.75], tf.convert_to_tensor([
  0, 0.5, 1])) -> tf.Tensor([[0., 0.5, 0.5]], shape(1, 3), dtype=float32).